pkg/ovs/ovsctl/ofctl.go

```diff
@@ -120,7 +121,15 @@ func (c *ovsCtlClient) DumpPortsDesc() ([][]string, error) {
 
 func (c *ovsCtlClient) SetPortNoFlood(ofport int) error {
 	cmdStr := fmt.Sprintf("ovs-ofctl mod-port %s %d no-flood", c.bridge, ofport)
-	return getOVSCommand(cmdStr).Run()
+	cmd := getOVSCommand(cmdStr)
+
+	var stderr bytes.Buffer
+	cmd.Stderr = &stderr
+	err := cmd.Run()
+	if err != nil {
+		return fmt.Errorf("error: %v, stderr: %s", err, string(stderr.Bytes()))
```

Comment:
What we will be the error message? Would you add information about setting no flood?

Reply:
Done. Change to:
```
return fmt.Errorf("fail to set no-food config for port %d on bridge %s: %v, stderr: %s", ofport, c.bridge, err, string(stderr.Bytes()))
```